packages/core/test/instrumentation.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @returns true if target[method] is instrumented by action. (e.g. via
+ * instrumentMethod())
+ */
+export function instrumentationIsAddedTo<TARGET extends { [key: string]: any }, METHOD extends keyof TARGET & string>(
```

Comment:
This helper is super trivial with the existing version of `instrumentMethod()`, but it's nice to have it, because if we rework `instrumentMethod()` to use a longer-lived observer-like pattern, the equivalent condition becomes a little trickier to write. The updated version of #3214 will replace the version you see here with a new one that handles the new `instrumentMethod()` implementation added in that PR.